importer-mankeli/README.md

```diff
@@ -1,2 +1,13 @@
 # importer-mankeli
 Receives data through NATS and parses it into db
+
+# Overview of the data flow
+
+1. Mankeli subscribes to Redis listener and listens on the CURRENT_EXECUTION_HASH channel.
+2. Once it receives the message on this channel, it knows that the importer-api started the update process, so it subscribes to all the "service" channels.
+3. When it receives a message on one of the service channels, it splits the received entities into two arrays: active and deleted ones.
+4. It then bulk upserts the active entities and bulk deletes the deleted ones.
```

Comment:
> bulk deletes the deleted ones.

This is not the case always.
Many entities such as studyRights, organisations and enrolments are actually not deleted, but instead the deleted entities are also bulk upserted, they then just have state="DELETED". Could add some caveat here.

Reply:
Its done like that for some entities because of some services using importer. For example, the information that an enrolment has been deleted must be propagated to Norppa so it can delete the corresponding "enrolment" from its db.

Its annoying that its done inconsistently for different entities but its just depending on the needs of the services.

Reply:
The exceptions could/should be listed? And perhaps also the reason why it must remain

Reply:
Got you, I'll add a note about this